build_base_lxc.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+cd $(dirname $(realpath $0))
+source "./lib/common.sh"
+
+function rebuild_base_lxc()
+{
+    check_lxd_setup
+
+    set -x
+    sudo lxc info $LXC_BASE >/dev/null && sudo lxc delete $LXC_BASE --force
+    sudo lxc launch images:debian/$DIST/$ARCH $LXC_BASE
+    sudo lxc config set $LXC_BASE security.privileged true
+    sudo lxc config set $LXC_BASE security.nesting true # Need this for apparmor for some reason
+    sudo lxc restart $LXC_BASE
+    sleep 5
+    
+    IN_LXC="sudo lxc exec $LXC_BASE --"
+    
+    INSTALL_SCRIPT="https://install.yunohost.org/$DIST"
+    $IN_LXC apt install curl -y
+    $IN_LXC /bin/bash -c "curl $INSTALL_SCRIPT | bash -s -- -a $YNH_BRANCH"
```

Comment:
`$YNH_BRANCH` is not defined

Reply:
Indeed .. that was supposed to be https://github.com/YunoHost/package_check/blob/cleanup-3/lib/common.sh#L7 but forgot to rename stuff ... Ideally also should add a small tweak to automatically prefix `-d` if that var is not empty

Reply:
we can use something like:

```
YNH_BRANCH=${1:-stable}
```